README.md

```diff
@@ -658,17 +658,25 @@ This macro returns a dictionary from a sql query, so that you don't need to inte
 
 **Usage:**
 ```
--- Returns a dictionary of the users table where the state is California
-{% set california_cities = dbt_utils.get_query_results_as_dict("select * from" ~ ref('cities') ~ "where state = 'CA' and city is not null ") %}
+{% set sql_statement %}
+    select city, state from {{ ref('users) }}
+{% endset %}
+
+{%- set places = dbt_utils.get_query_results_as_dict(sql_statement) -%}
+
 select
-  city,
-{% for city in california_cities %}
-  sum(case when city = {{ city }} then 1 else 0 end) as users_in_{{ city }},
-{% endfor %}
-  count(*) as total
-from {{ ref('users') }}
 
-group by 1
+    {% for city in places['CITY'] | unique -%}
+      sum(case when city = '{{ city }}' then 1 else 0 end) as users_in_{{ city | replace(' ', '_') }},
```

Comment:
What about using [slugify](https://github.com/dbt-labs/dbt-utils#slugify-source) instead?

Reply:
I think it's OK for this basic example to pretend that no users are from Martha's Vineyard, right? 

Related: while writing this comment, I [discovered](https://forum.wordreference.com/threads/apostrophe-in-place-names.2628789/post-13258411) that only five place names in the US are allowed to contain an apostrophe!

Reply:
I didn't know about slugify~! Love the suggestion, added an update. Also fixed the slugify example since it was missing the package prefix 🥲